corehq/sql_accessors/migrations/0059_get_location_fixture_ids.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-03-15 18:43
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {})
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0058_drop_old_reverse_indexed_fn'),
+    ]
+
+    operations = [
+        migrator.get_migration('get_location_fixture_ids.sql'),
```

Comment:
i think the sql_accessors migrations only get run in the sharded dbs. 

https://github.com/dimagi/commcare-hq/blob/master/corehq/sql_db/routers.py#L70-L71

Reply:
Heh, wonder how the tests passed? OK, I'll move it to a different app in that case.

Reply:
i'm guessing it didn't run in any tests that use the partitioned setup. so in that case everything just uses the main db and it would work

Reply:
https://github.com/dimagi/commcare-hq/pull/19826/commits/3012999c692b58029790662b38d5ce597b8fee68